### DISCUSSION TITLE
Add support for fanatec devices

### DISCUSSION THREAD

**Discussion by tim-gromeyer** [fwupd/fwupd#6774] [General]
**Is your feature request related to a problem? Please describe.**
Not really, I just want to update the firmware of my fanatec devices (CSL DD Pro, Clubsport shifter, Load cell kit) and my Windows VM doesn not work for this, because USB device pass though is somehow broken

**Describe the solution you'd like**
-

**Describe alternatives you've considered**
-

**Additional context**

**Comment by hughsie** [comment]
Do you have any contacts at fanatec we could ask?

**Reply by tim-gromeyer** [reply]
No, unfortunately not. I can ask the contact, I spend 1200€ for their products, I think I can get some help from them

**Reply by hughsie** [reply]
Okay, if you do find out anything, my email is richard_at_hughsie_dot_com -- feel free to include me in any conversations and I can jump in with details.
